plugins/org.eclipse.sirius.debug/src/org/eclipse/sirius/api/debug/AbstractDebugView.java

```diff
@@ -77,7 +74,7 @@ public Action(String label, String description, Runnable body) {
     /**
      * The currently selection object.
      */
-    protected Object selection;
+    public Object selection;
 
     @Override
     public void createPartControl(Composite parent) {
```

Comment:
Checkstyle rule: for the above line checkstyle must be disabled:
```
        // CHECKSTYLE:OFF
        info = new Text(parent, SWT.MULTI | SWT.READ_ONLY | SWT.WRAP | SWT.BORDER | SWT.H_SCROLL | SWT.V_SCROLL);
        // CHECKSTYLE:ON
```

Reply:
done